mmv1/products/compute/InterconnectGroup.yaml

```diff
@@ -0,0 +1,248 @@
+          of the Interconnect product SLAs.
+        enum_values:
+          - 'PRODUCTION_NON_CRITICAL'
+          - 'PRODUCTION_CRITICAL'
+          - 'NO_SLA'
+          - 'AVAILABILITY_SLA_UNSPECIFIED'
+    required: true
+  - name: 'physicalStructure'
+    type: NestedObject
+    description: |
+      An analysis of the physical layout of Interconnects in this
+      group. Every Interconnect in the group is shown once in this structure.
+    properties:
+      - name: 'metros'
+        type: Array
+        description: |
+          The metros Interconnects in this group are in.
+        item_type:
+          type: NestedObject
+          properties:
+            - name: 'metro'
+              type: String
+              description: |
+                The name of the metro, as a three-letter lowercase string
+                like "iad". This is the first component of the location of
+                Interconnects underneath this.
+            - name: 'facilities'
+              type: Array
+              description: |
+                The facilities Interconnects in this metro are present
+                in.
+              item_type:
+                type: NestedObject
+                properties:
+                  - name: 'facility'
+                    type: String
+                    description: |
+                      The ID of this facility, as a numeric string like
+                      "5467". This is the third component of the location of Interconnects
+                      in this facility.
+                  - name: 'zones'
+                    type: Array
+                    description: |
+                      The zones that Interconnects in this facility are
+                      present in.
+                    item_type:
+                      type: NestedObject
+                      properties:
+                        - name: 'zone'
+                          type: String
+                          description: |
+                            The name of the zone, either "zone1" or "zone2".
+                            This is the second component of the location of Interconnects in
+                            this facility.
+                        - name: 'interconnects'
+                          type: Array
+                          description: |
+                            URLs of Interconnects in this redundancy group in the
+                            given metro, facility, and zone.
+                          item_type:
+                            type: String
+    output: true
+  - name: 'configured'
+    type: NestedObject
+    description: |
+      The status of the group as configured. This has the same
+      structure as the operational field reported by the OperationalStatus
+      method, but does not take into account the operational status of each
+      resource.
+    properties:
+      - name: 'topologyCapability'
```

Comment:
Same thing with `output: true` again (and again `supportedSla` and `blockerType` are not documented as output only but sound like they probably are)